rotors_gazebo_plugins/cmake/FindGlog.cmake

```diff
@@ -0,0 +1,155 @@
+# Ceres Solver - A fast non-linear least squares minimizer
```

Comment:
Are you sure you need this cmake file? shouldn't adding `<build_depend>libgoogle-glog-dev</build_depend>` in the `package.xml` fix this?